test/test_models.py

```diff
@@ -88,6 +98,7 @@ def _test_detection_model(self, name):
         out = model(model_input)
         self.assertIs(model_input[0], x)
         self.assertEqual(len(out), 1)
+        self.assertExpected(out, name)
```

Comment:
` maskrcnn_resnet50_fpn_expect.pkl ` is kind of large here - `+34.3 MB` maybe we could check something like sum and variance here instead of the actual tensor value ?

Reply:
Yeah, 34MB is a bit large. One thing that @fbbradheintz is doing in https://github.com/pytorch/vision/pull/1373 is to select a subset of the output elements. He has done this for classification models for now, not sure what his thoughts are about detection models

Reply:
I started with detection models sort of arbitrarily, mostly just in anticipation of https://github.com/pytorch/vision/pull/1333. I don't see any reason why the same approach shoudn't apply to other types of models.

Agreed 34MB is too large. Current ideas are:
- moments of the results. we can also pass in a custom tolerance to `assertExpected` if we think it will be too fragile.
- subset of the output elements. One suggestion to do this is by flattening all tensor outputs, and then taking every random `nth` index such that the resulting tensor isn't more than `X` number of elements (100?).
- both ? this would be more robust to sparse tensors  
 
 
@fbbradheintz what do you think?

Reply:
I'm a fan of the sampling method, though I may change the sampling number to be a prime number an order of magnitude or two less than the number of output elements, just to avoid the possibility of alignment artifacts.

Reply:
Actually, looking over code again, that's not going to matter for one-dimensional classification outputs. I'd still be wary of alignment issues on multi-dimensional outputs.

Reply:
about multi-dimensional tensors, I'd just check that the shape of the tensors are also the same, and then we should be good to go?